packages/form/MIGRATION.md

```diff
@@ -0,0 +1,117 @@
+# Migrating from `availity-reactstrap-validation` to `@availity/form`
+
+The primary difference in developing with `availity-reactstrap-validation` and `@availity/form` is validation. With `availity-reactstrap-validation`, we define validation schemas on the input components in the form with the `validate` prop, like so:
+
+```jsx
+import { AvInput, AvForm } from 'availity-reactstrap-validation';
+
+// ...
+<AvForm>
+  <AvInput
+    name="myInput"
+    type="text"
+    validate={{
```

Comment:
to make this more realistic this would be:
```
<AvInput name="myInput" required />
```

Reply:
My goal is to bring out the comparison of the `validate` prop in `availity-reactstrap-validation` and the `validationSchema` prop in `@availity/form`. So, yes just the `required` prop is probably how must people are using that library, but it makes the comparison less clear. Your comment makes me think we should add a note on both the `required` prop _and_ the `validate` prop.

Reply:
d803ec1527dd3b76c881aa771a61964fac6cc1dc